packages/core/modules/config/default.js

```diff
@@ -24,6 +24,7 @@ export const settings = {
   canDeleteLocked: false,
   canLeaveEmptyGroup: true,
   shouldCreateEmptyGroup: false,
+  jsonLogicExportPreserveGroups: false,
```

Comment:
Please rename to more generic name like `exportPreserveGroups`
Same approach could be used for other formats.